public_dropin_environments/java_codegen/dr_requirements.txt

```diff
@@ -1,3 +1,3 @@
 pandas==1.0.5
 pyarrow==0.14.1
-datarobot-drum==1.6.2
+datarobot-drum==1.6.2.post1
```

Comment:
didnt notice its a dot. I think we thought of "1.6.2-post1"

@meltzg will this break anything?

Reply:
no, it won't
When I use `-` python normalizes it to dot in the filename

Reply:
what do you normalized?

Reply:
https://www.python.org/dev/peps/pep-0440/#post-release-separators

it can be installed with either:
pip install datarobot-drum==1.6.2.post1
pip install datarobot-drum==1.6.2-post1